src/components/button/button.scss

```diff
@@ -65,5 +67,8 @@ button {
         svg {
             opacity: 1;
         }
+        limel-spinner {
+            display: block;
+        }
```

Comment:
With this addition under `.just-loaded`, the spinner properly fades out when loading ends, but is properly hidden 2 seconds later, when the checkmark disappears.

Reply:
awesome!